rattan-core/src/cells/delay.rs

```diff
@@ -443,13 +496,13 @@ where
             egress: DelayReplayCellEgress {
                 egress: tx,
                 trace,
-                current_delay: Delay::ZERO,
-                next_available: Instant::now(),
+                delays: Configs::new(),
                 next_change: Instant::now(),
                 config_rx,
                 send_timer: Timer::new()?,
                 change_timer: Timer::new()?,
-                state: AtomicI32::new(0),
+                state: AtomicState::new(State::Drop),
+                latest_egress_timestamp: Instant::now() - Duration::from_secs(10),
```

Comment:
The same as above, this maybe a potential bug.

Reply:
And as we use the 10 many times, we should define the Duration::new(10,0) as a const var.